### PR TITLE
Unify logger usage

### PR DESCRIPTION
**Pull request by eunwoosh**
### Summary
Current OTX uses different logger per files. This PR unified it to use same OTX logger.
Changes this PR made are as below:

- Change part using other logging method to use OTX logging method.
- To avoid circular import error, logger file is moved to `otx/utils`.
- After config manager instance is made, `config_logger` is called in each cli files.
 

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: src/otx/algorithms/classification/adapters/mmcls/utils/builder.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from mmcv.runner import load_checkpoint
 from mmcv.utils import Config, ConfigDict, get_logger
 
-from otx.algorithms.common.utils.logger import LEVEL
+from otx.utils.logger import LEVEL
 
 logger = get_logger("mmcls")
```
Comment (jaegukhyun) [inline src/otx/algorithms/classification/adapters/mmcls/utils/builder.py]: Is it okay to use get_logger function like this?

Comment (eunwoosh) [reply src/otx/algorithms/classification/adapters/mmcls/utils/builder.py]: Thanks for finding. I missed it. I'll update.

Comment (eunwoosh) [reply src/otx/algorithms/classification/adapters/mmcls/utils/builder.py]: fixed at [4051d39](https://github.com/openvinotoolkit/training_extensions/pull/2612/commits/4051d395412953637667133bfea8f918151161c9)

**Review by jaegukhyun — APPROVED** [review]

File: src/otx/cli/tools/train.py [hunk 2]
```diff
@@ -199,6 +200,7 @@ def train(exit_stack: Optional[ExitStack] = None):  # pylint: disable=too-many-b
     args, override_param = get_args()
 
     config_manager = ConfigManager(args, workspace_root=args.workspace, mode=mode)
+    config_logger(config_manager.output_path / "otx.log", "INFO")
```
Comment (sungmanc) [inline src/otx/cli/tools/train.py:203]: How about adding the detailed path? If user conduct `train->eval->export->eval`, then it seems that user can't check the train log`. Also same opinion for the eval, explain, ...
```suggestion
    config_logger(config_manager.output_path / "otx_train.log", "INFO")
```

Comment (eunwoosh) [reply src/otx/cli/tools/train.py:203]: log file is made in each output directory as below. I think it's unnecessary to have different name. How do you think about?
image

Comment (sungmanc) [reply src/otx/cli/tools/train.py:203]: Ah, if so I don't have any concerns :)

**Review by sungmanc — APPROVED** [review]

**Pull request merged**